todo-app/src/App.js

```diff
@@ -1,13 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
+import {AppTask} from './AppTask';
+import {AppDate} from './AppDate';
 
-class App extends Component {
-  render() {
-    return (
-      <div>
-        Hello World!
-      </div>
-    );
-  }
+export class Todo extends React.Component{
+    Constructor(){
```

Comment:
I guess you must have been looking at React component examples somewhere? For this homework, because we don't have any component state or functions, we don't need a constructor.

We will get more into functions and state this coming week. Just know that a class `constructor` is _not necessary_ for a React component, but if you do choose to use it, it's lowercase: `constructor`.

Reply:
hahaha yes you are right it was a tutorial on youtube!!